driver-sync/src/test/functional/com/mongodb/client/ReadConcernTest.java

```diff
@@ -68,18 +69,16 @@ public void cleanUp() {
     @Test
     public void shouldIncludeReadConcernInCommand() {
         mongoClient.getDatabase(getDefaultDatabaseName()).getCollection("test")
-                .withReadConcern(ReadConcern.LOCAL).estimatedDocumentCount();
+                .withReadConcern(ReadConcern.LOCAL).find().into(new ArrayList<>());
```

Comment:
No Async equivalent?

Reply:
There is a ReadConcernTest in reactive streams module that was also modified.